napari_allencell_segmenter/core/layer_reader.py

```diff
@@ -37,13 +37,18 @@ def get_channels(self, layer: Layer) -> List[Channel]:
         return self._get_channels_default(layer)
 
     def _get_channels_default(self, layer: Layer) -> List[Channel]:
-        img = AICSImage(layer.data)  # gives us a 6D image
+
+        if len(layer.data.shape) >= 6:
+            # Has scenes
+            image_from_layer = [layer.data[i, :, :, :, :, :] for i in range(layer.data.shape[0])]
+        else:
+            image_from_layer = layer.data
+        img = AICSImage(image_from_layer)  # gives us a 6D image#
+        img.set_scene(0)
```

Comment:
How do we know to use 0?

Reply:
We're just using the first scene. Same thing as before, instead we do it this way now instead of feeding a kwarg `s=0` into the img.get_image_data() function